client/layout/masterbar/checkout.tsx

```diff
@@ -54,21 +55,34 @@ const CheckoutMasterbar = ( {
 	const { responseCart, replaceProductsInCart } = useShoppingCart( cartKey );
 	const [ isModalVisible, setIsModalVisible ] = useState( false );
 
-	const closeAndLeave = ( options?: { userHasClearedCart?: boolean } ) =>
+	const closeAndLeave = ( options?: {
+		userHasClearedCart?: boolean;
+		closedWithoutConfirmation?: boolean;
+	} ) => {
+		const userHasClearedCart = options?.userHasClearedCart ?? false;
+		if ( ! options?.closedWithoutConfirmation ) {
+			recordTracksEvent( 'calypso_masterbar_checkout_close_modal_submitted', {
```

Comment:
Oh, nice that there's a way to do this without needing Redux, unlike [the other `recordTracksEvent`](https://github.com/Automattic/wp-calypso/blob/50ef2f4a0aefa3c131653b5be607425ffb59759c/client/my-sites/checkout/src/components/empty-cart.tsx#L15).

Reply:
I guess there is no way to detect if the back button was used to "close" Checkout? Could we track navigating away from Checkout as a "bounce"?

![image](https://github.com/user-attachments/assets/fb30dd12-db89-4100-aa9f-cb365d068fa1)